src/project/internal/projectfilescontroller.cpp

```diff
@@ -486,9 +486,11 @@ io::path ProjectFilesController::selectScoreOpeningFile()
            << QObject::tr("MuseScore Dev Files") + " (*.mscs)"
            << QObject::tr("MuseScore Backup Files") + " (*.mscz~)";
 
-    return interactive()->selectOpeningFile(qtrc("project", "Score"), "", filter.join(";;"));
+    return interactive()->selectOpeningFile(qtrc("project", "Score"), configuration()->userProjectsPath() , filter.join(";;"));
```

Comment:
Oh, one more thing (sorry), I see here's an extra space before the comma, and probably the code style test will not allow that.

Reply:
yeah,I fixed that in my latest commit.